debezium-history-compaction-tool/src/test/java/io/debezium/compaction/tool/service/compaction/KafkaDatabaseHistoryCompactionTest.java

```diff
@@ -60,20 +64,58 @@ public static void stopKafka() {
         }
     }
 
-    @Test
-    public void testConfiguration() throws InterruptedException {
-        if (kafka != null) {
-            System.out.println("is kafka running: " + kafka.isRunning());
+    @Before
+    public void beforeEach() {
+        MySqlPartition source = new MySqlPartition("dbserver", "my-db");
+        this.config = Configuration.empty().edit().with(RelationalDatabaseConnectorConfig.SERVER_NAME, "dbserver").build();
+
+        position = new MySqlOffsetContext(false, true, new TransactionContext(), new MySqlReadOnlyIncrementalSnapshotContext<>(),
+                new SourceInfo(new MySqlConnectorConfig(config)));
+
+        setLogPosition(0);
+        offsets = Offsets.of(source, position);
+
+    }
+
+    @After
+    public void afterEach() {
+        try {
+            if (historyCompaction != null) {
+                historyCompaction.stop();
+            }
         }
-        else {
-            System.out.println("Kafka is not running it's null");
+        finally {
+            historyCompaction = null;
         }
+    }
 
-        // KafkaDatabaseHistoryCompaction databaseHistoryCompaction = new KafkaDatabaseHistoryCompaction("", "", "");
-        //
-        // Configuration dbHistoryConfig = Configuration.create().build();
-        //
-        // databaseHistoryCompaction.configure(dbHistoryConfig, null, DatabaseHistoryListener.NOOP, false);
+    @Test
+    public void shouldStartWithEmptyHistoryTopic() {
+        final String historyTopicName = "my-db-history-topic";
+
+        kafka.createTopic(historyTopicName, 1, 1);
+
+    }
```

Comment:
Hi @jatinderaujla, all this test does is creates the topic.  As we discussed earlier in the week, the goal of this is to not only create the empty topic, but to also run the compaction tool against it and verify that the compaction tool runs without a failure if the source topic is empty.  You aren't running the compaction tool here from what i see.